listenbrainz/webserver/views/feedback_api.py

```diff
@@ -0,0 +1,125 @@
+import ujson
+import listenbrainz.db.user as db_user
+import listenbrainz.db.feedback as db_feedback
+
+from datetime import datetime
+from flask import Blueprint, current_app, jsonify, request
+from listenbrainz.webserver.decorators import crossdomain
+from listenbrainz.webserver.errors import (APIBadRequest,
+                                           APIInternalServerError, APINotFound,
+                                           APIServiceUnavailable,
+                                           APIUnauthorized)
+from listenbrainz.webserver.rate_limiter import ratelimit
+from listenbrainz.webserver.views.api import _validate_auth_header, _parse_int_arg
+from listenbrainz.webserver.views.api_tools import log_raise_400, is_valid_uuid
+from listenbrainz.feedback import Feedback
+from pydantic import ValidationError
+
+feedback_api_bp = Blueprint('feedback_api_v1', __name__)
+
+
+@feedback_api_bp.route("submit-feedback", methods=["POST", "OPTIONS"])
```

Comment:
`/1/feedback/submit-feedback`? I'm wondering if this should be recording feedback? Feedback could refer to many things.

Reply:
Updated to /1/feedback/recording-feedback